include/tvm/relay/expr.h

```diff
@@ -486,6 +486,18 @@ std::string RelayPrint(
     const NodeRef& node,
     bool show_meta_data = true,
     runtime::TypedPackedFunc<std::string(Expr)> annotate = nullptr);
+
+/*!
+ * \brief Print node as text format. This function prints AST nodes only and
+ *        the metadata is omitted.
+ * \param node The node to be printed.
+ * \param annotate An optional callback function for attaching
+ *        additional comment block to an expr.
+ * \return The text representation.
+ */
+std::string Print(const NodeRef& node,
```

Comment:
Let us simply overload operator<< for relayExpr in this case.

Reply:
@tqchen Yeah, I tried overloading before, but there is already an overloaded one here: https://github.com/dmlc/tvm/blob/master/src/lang/expr.cc#L46 So I simply used Print.

Reply:
@tqchen Sorry, you were saying Expr. That should work. I will update the PR.

Reply:
Are we sure we want to have default be the pretty-printed version of the program? It is valuable to the debug representation as well, how do we do that?

Reply:
@jroesch I agree that metadata can be useful for debugging, but I think pretty-printing could probably provide enough information most of the time if I understand correctly. People can still have the detailed information using the previous way for debugging if needed.
@tqchen @ZihengJiang thought?

Reply:
If you feel that there is a need to avoid c++ printing for now. we can first remove the operator<< in C++ and only land the python side of changes